docs/data-backlog.md

```diff
@@ -0,0 +1,133 @@
+| Last updated at | Date | When was the data last updated / refreshed? We need a way to capture recency if not able to get a date |
+| Frequency of updates | Text | How often is this data updated? Is it updated on a reliable cadence? |
+| Peer review | Text | Overview or links out to peer review done on this dataset |
+| Where/how is data available | Text | Is it available on Geoplatform.gov? Is it available from multiple sources? Do we need to SFTP into a server to get it? (Maybe this should be two separate columns) |
+| Category / taxonomy keywords | Array | Since the taxonomy for these indicators and datasets isn't an exact science and different audiences may use different ways to refer to the data set categories, let's use a list of keywords instead of a single word |
+| Accreditation | Text | Is this source accredited? |
+| Can data go in cloud? | Yes/No | Some datasets can not legally go in the cloud |
+| Documentation | Text / URL | Link to docs. Also, is the documentation good enough? Can we get the info we need? |
+| Legal weasels | Text | Legal disclaimers, assumption of risk, proprietary? |
+| Risk assessment of the data | Text or High/Medium/Low | E.g. a vendor-processed version of the dataset might not be open or good enough |
+
+Other ideas for consideration:
+
+* Transparency
+* Reproducibility
+* A way to capture the owner vs host of the data
+* How likely are the data owner/host is to be able to update the data at proposed frequency
+
+# Tooling proposal and milestones
+
+There is no single tool that supports all the goals and user stories above. Therefore we've proposed combining a number of tools in a way that can support them all.
+
+We've also proposed various "milestones" that will allow us to iteratively and sequentially build the data backlog in a way that supports the entire vision but starts with small and achievable steps. These milestones are proposed in order.
+
+## Milestone: YAML files for data sets and linter
+
+To start, we'll create a folder in the appropriate repository [is this `justice40-tool`?] that can house YAML files, one per data set. Each file will describe the characteristics of the data.
+
+The benefit of using a YAML file for this is that it's easy to subject changes to these files to peer review through the pull request process. This allows external collaborators from the open source community to submit suggested changes, which can be reviewed by the core CEJST team.
+
+We'll use a Python-based script to load all the files in the directory, and then run a schema validator to ensure all the files have valid entries.
+
+For schema validation, we propose using [Marshmallow](https://marshmallow.readthedocs.io/en/stable/). This is slightly heavier-weight than necessary, but since we're planning to use Marshmallow as a serialization library elsewhere in the project, it helps to keep things consistent.
```

Comment:
I'm thinking we can do this without an explicit tool to process - if we use something like [Yamale](https://github.com/23andMe/Yamale) we don't have to maintain a tool ourselves; there's even a Github Actions integration [here](https://github.com/nrkno/yaml-schema-validator-github-action)